api/MessageEvent.json

```diff
@@ -403,10 +403,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "4"
             },
             "safari_ios": {
-              "version_added": "≤3"
+              "version_added": "3"
```

Comment:
The data for `postMessage` says 3.2. It would make sense for `MessageEvent` and `postMessage` to be introduced together, can you check which is correct?

Reply:
This was a mirroring error, whoops!  Fixed!